pkg/config/profile_test.go

```diff
@@ -101,6 +104,43 @@ test_mode_key_expires_at = '` + expiresAt + `'
 	cleanUp(c.ProfilesFile)
 }
 
+func TestAPIKeyLogLevel(t *testing.T) {
+	// Set the level to debug
+	logrus.SetLevel(logrus.DebugLevel)
+
+	c := &Config{
+		Color:    "auto",
+		LogLevel: "debug",
+		Profile: Profile{
+			ProfileName:    "tests",
+			TestModeAPIKey: "asdas",
+		},
+		ProfilesFile: "",
+	}
+
+	// For debug mode, the error should complain about a config file missing
+	// since we did not init the config
+	key, err := c.Profile.GetAPIKey(false)
+
+	// Little weird but windows returns a different error message than others
+	os := runtime.GOOS
+	switch os {
+	case "windows":
+		assert.ErrorContains(t, err, `config.toml: The system cannot find the file specified.`)
+	default:
+		assert.ErrorContains(t, err, `config.toml: no such file or directory`)
```

Comment:
Sometimes these error messages change or vary (e.g., different linux distros) and I'd like us to have a very obvious and actionable signal if/when that happens.

Can we have an assertion somewhere that alerts us if neither of the two cases we know about today are met?